lib/buildkite/test_collector/ci.rb

```diff
@@ -31,13 +31,14 @@ def analytics_env
       "url" => ENV["BUILDKITE_ANALYTICS_URL"],
       "branch" => ENV["BUILDKITE_ANALYTICS_BRANCH"],
       "commit_sha" => ENV["BUILDKITE_ANALYTICS_SHA"],
-      "number" => ENV["BUILDKITE_ANALYTICS_NUMBER"],
       "job_id" => ENV["BUILDKITE_ANALYTICS_JOB_ID"],
       "message" => ENV["BUILDKITE_ANALYTICS_MESSAGE"],
       "execution_name_prefix" => ENV["BUILDKITE_ANALYTICS_EXECUTION_NAME_PREFIX"],
       "execution_name_suffix" => ENV["BUILDKITE_ANALYTICS_EXECUTION_NAME_SUFFIX"],
       "version" => Buildkite::TestCollector::VERSION,
       "collector" => "ruby-#{Buildkite::TestCollector::NAME}",
+      "build_number" => ENV["BUILDKITE_ANALYTICS_NUMBER"],
+      "build_id" => ENV["BUILDKITE_BUILD_ID"]
```

Comment:
This one is used for overriding the default values and differs from the baked-in `BUILDKITE_BUILD_ID` it should be:
```suggestion
      "build_id" => ENV["BUILDKITE_ANALYTICS_BUILD_ID"]
```
This also makes it possible for someone using CI other than the Buildkite, GitHub Actions or CircleCI to pass in an equivalent value from their environment without overloading the meaning of the env set by Buildkite Pipelines.

Reply:
I've implemented this, but I can't find `BUILDKITE_ANALYTICS_BUILD_ID` anywhere in the docs, is this a new variable we need to add?